### PR TITLE
Fix running bridge in docker-compose

### PR DESCRIPTION
**Pull request by DmitriyMolch**

### CODE REVIEW

File: config/config.json.example [hunk 1]
```diff
@@ -0,0 +1,54 @@
+{
+  "chains": [
+    {
+      "name": "eth",
+      "type": "ethereum",
+      "id": "0",
+      "endpoint": "https://goerli.infura.io/v3/178b87e860404572bc217ca53fe594bd",
```
Comment (AndreiNavoichyk) [inline config/config.json.example]: Public URL should be used. Please, update

File: config/config.json.example [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+        "startBlock": "6455184",
+        "blockConfirmations": "12",
+        "http": "true"
+      }
+    },
+    {
+      "name": "polygon",
+      "type": "ethereum",
+      "id": "2",
+      "endpoint": "https://polygon-mumbai.infura.io/v3/178b87e860404572bc217ca53fe594bd",
```
Comment (AndreiNavoichyk) [inline config/config.json.example]: Public URL should be used. Please, update

File: docker-compose.yml [hunk 3]
```diff
@@ -4,12 +4,13 @@
 version: '3'
 services:
   bridge:
+    user: "${UID}:${GID}"
```
Comment (AndreiNavoichyk) [inline docker-compose.yml:7]: Why do we need this?

File: README.md [hunk 4]
```diff
@@ -179,6 +164,43 @@ MKdocs will generate static HTML files for Chainsafe markdown files located in `
...
+``` 
+docker-compose up
+```
+## Run app from source code:
+
+1. Export password:
+```
+export KEYSTORE_PASSWORD=12345678
+```
+2. Run executable:
```
Comment (AndreiNavoichyk) [inline README.md:204]: Add step to build executable before this step.

Comment (AndreiNavoichyk) [inline README.md:204]: Remove executable from the repo.

File: README.md [hunk 5]
```diff
@@ -179,6 +164,43 @@ MKdocs will generate static HTML files for Chainsafe markdown files located in `
 
 `make mkdocs`: Run MkDoc's docker image, building and hosting the html files on `localhost:8000`  
 
+# Local run
+## Prepare:
+
+1. Import keys:
```
Comment (AndreiNavoichyk) [inline README.md]: Add that app will connect to Devnet.

File: docker-compose.yml (hunk 3 above)
Comment (DmitriyMolch) [reply docker-compose.yml:7]: To be able to read the key file

File: README.md (hunk 4 above)
Comment (DmitriyMolch) [reply README.md:204]: Why don't we build it to the `./build` folder as it is written in line 66 in README? Can we change it?

File: config/config.json.example (hunk 1 above)
Comment (DmitriyMolch) [reply config/config.json.example]: fixed

File: config/config.json.example (hunk 2 above)
Comment (DmitriyMolch) [reply config/config.json.example]: fixed

File: README.md (hunk 4 above)
Comment (DmitriyMolch) [reply README.md:204]: fixed

Comment (DmitriyMolch) [reply README.md:204]: fixed

Comment (AndreiNavoichyk) [reply README.md:204]: I see. The build step is in place. Let's reuse the executable from it.
Why do we need to change it? `build` directory looks good.

File: README.md [hunk 6]
```diff
@@ -179,6 +164,46 @@ MKdocs will generate static HTML files for Chainsafe markdown files located in `
...
+```
+3. Copy `config/config.json.example` to `config/config.json` folder:
+```
+cp config/config.json.example config/config.json
+```
+
+## Run app in docker:
+
+``` 
+docker-compose up
```
Comment (AndreiNavoichyk) [inline README.md:195]: Why do we need to build an executable before the `docker-compose up instruction? It should build everything by its own

File: README.md (hunk 4 above)
Comment (DmitriyMolch) [reply README.md:204]: I just updated it to avoid it calling it from `cmd` folder

File: README.md (hunk 6 above)
Comment (DmitriyMolch) [reply README.md:195]: Because we need to create keys with executable

File: config/config.json.example (hunk 2 above)
Comment (AndreiNavoichyk) [reply config/config.json.example]: I still see old link

File: config/config.json.example (hunk 1 above)
Comment (AndreiNavoichyk) [reply config/config.json.example]: I still see old link

File: README.md [hunk 7]
```diff
@@ -179,6 +164,46 @@ MKdocs will generate static HTML files for Chainsafe markdown files located in `
 
 `make mkdocs`: Run MkDoc's docker image, building and hosting the html files on `localhost:8000`  
 
+# Local run
+## Prepare:
+1. Build executable:
+```
```
Comment (AndreiNavoichyk) [inline README.md]: Please, add a tab space before all content under the numbered list items so it looks visually as inner

File: README.md [hunk 8]
```diff
@@ -179,6 +164,46 @@ MKdocs will generate static HTML files for Chainsafe markdown files located in `
...
+cp config/config.json.example config/config.json
+```
+
+## Run app in docker:
+
+``` 
+docker-compose up
+```
+## Run app from source code:
+
```
Comment (AndreiNavoichyk) [inline README.md:199]: Please, use the same formatting everywhere. In the line 169 you don't have empty line.

File: README.md [hunk 9]
```diff
@@ -179,6 +164,46 @@ MKdocs will generate static HTML files for Chainsafe markdown files located in `
...
+build/chainbridge accounts import --privateKey "improve nominee response kangaroo keen gain antenna pepper spike credit pony parrot" --sr25519
+```
+```
+build/chainbridge accounts import --privateKey 5502ab637600e552b45d0608f8f2888a5feafa2a5df1bc67eb9ecc69a9c6b990
+```
+```
+build/chainbridge accounts import --privateKey a5485b158e045febc939d9cedc79910f26efe48c31f65643fba3e6dd0f734282
+```
+```
+Enter a password after a prompt appears: 12345678
```
Comment (AndreiNavoichyk) [inline README.md]: Let's write that any password can be used and `12345678` can be used for example.

File: README.md [hunk 10]
```diff
@@ -179,6 +164,46 @@ MKdocs will generate static HTML files for Chainsafe markdown files located in `
 
 `make mkdocs`: Run MkDoc's docker image, building and hosting the html files on `localhost:8000`  
 
+# Local run
+## Prepare:
+1. Build executable:
+```
+`make build`
+```
+2. Import keys from devnet:
```
Comment (AndreiNavoichyk) [inline README.md]: Please, use capital letters for network names. **D**evnet

File: README.md (hunk 5 above)
Comment (DmitriyMolch) [reply README.md]: Already fixed it. This is an old version of the file.

File: config/config.json.example (hunk 1 above)
Comment (DmitriyMolch) [reply config/config.json.example]: Fixed

File: config/config.json.example (hunk 2 above)
Comment (DmitriyMolch) [reply config/config.json.example]: Fixed

File: README.md (hunk 8 above)
Comment (DmitriyMolch) [reply README.md:199]: fixed

File: README.md (hunk 9 above)
Comment (DmitriyMolch) [reply README.md]: fixed

File: README.md [hunk 11]
```diff
@@ -179,6 +164,46 @@ MKdocs will generate static HTML files for Chainsafe markdown files located in `
...
+    Enter a password after a prompt appears: <any_password>
+    ```
+3. Copy `config/config.json.example` to `config/config.json` folder:
+    ```
+    cp config/config.json.example config/config.json
+    ```
+    
+## Run app in docker:
+
+    docker-compose up
```
Comment (AndreiNavoichyk) [inline README.md]: Please, wrap the command with code block

**Review by AndreiNavoichyk — APPROVED** [review]

File: README.md (hunk 11 above)
Comment (DmitriyMolch) [reply README.md]: I tried before and I produces extra symbols on top. I think this is how README syntax works.
image

Comment (DmitriyMolch) [reply README.md]: fixed

**Pull request merged**